src/main/java/org/broadinstitute/hellbender/tools/spark/sv/discovery/ChimericAlignment.java

```diff
@@ -191,9 +190,9 @@ static boolean firstAlignmentIsTooShort(final AlignmentInterval first, final Ali
      * To implement the idea that for two consecutive alignment regions of a contig, the one with higher reference coordinate might be a novel insertion.
      */
     @VisibleForTesting
-    public static boolean nextAlignmentMayBeInsertion(final AlignmentInterval current, final AlignmentInterval next,
-                                                      final Integer minAlignLength, final int mapQThresholdInclusive,
-                                                      final boolean filterWhollyContained) {
+    static boolean nextAlignmentMayBeInsertion(final AlignmentInterval current, final AlignmentInterval next,
```

Comment:
check how this is different from the new method `splitPairStrongEnoughEvidenceForCA()`